modules/cms/controllers/ThemeOptions.php

```diff
@@ -103,8 +107,12 @@ public function formExtendFieldsBefore($form)
 
     /**
      * Default to the active theme if user doesn't have access to manage all themes
+     *
+     * @param string $dirName
+     *
+     * @return string
      */
-    protected function getDirName($dirName = null)
+    protected function getDirName(string $dirName = null): string
```

Comment:
```suggestion
    protected function getDirName(string $dirName = null)
```
We don't really use return type hints a lot in October core.

Reply:
You'd better start using them 😄